File: fizzbuzz.py
# AH period 6 FuzzBazz - Python

for i in range(1, 51):
    # Is it divisible by 3 and 5
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    
    # Is it divisible by 3
    elif i % 3 == 0:
        print("Fizz")
    
    # Is it divisible by 5
    elif i % 5 == 0:
        print("Buzz")
    
    # If not divisible by 3 or 5
    else:
        print(i)

File: FizzBuzz.c
//AH period 6 FizzBuzz - C

#include <stdio.h>

int main() {
    // Loop from 1 to 50
    for (int i = 1; i <= 50; i++) {
        if (i % 15 == 0) {
            printf("FizzBuzz\n");
        } else if (i % 3 == 0) {
            printf("Fizz\n");
        } else if (i % 5 == 0) {
            printf("Buzz\n");
        } else {
            printf("%d\n", i);
        }
    }
    return 0;
}
